install/kubernetes/helm/istio/charts/mixer/templates/crds.yaml

```diff
@@ -432,6 +432,24 @@ spec:
   version: v1alpha2
 ---
 
+kind: CustomResourceDefinition
```

Comment:
@guptasu just to confirm, this is still where we are putting new templates, correct?